virttest/utils_test/libvirt.py

```diff
@@ -160,6 +161,25 @@ def cleanup(self):
             virsh.net_undefine(self.name)
 
 
+def create_macvtap_vmxml(iface, params):
+    """
+    Method to create Macvtap interface xml for VM
+
+    :param iface: macvtap interface name
+    :param params: Test dict params for macvtap config
+
+    :return: macvtap xml object
+    """
+    mode = params.get('macvtap_mode', 'passthrough')
+    model = params.get('macvtap_model', 'virtio')
+    macvtap_type = params.get('macvtap_type', 'direct')
+    macvtap = interface.Interface('direct')
```

Comment:
s/`macvtap = interface.Interface('direct')`/`macvtap = interface.Interface(macvtap_type)`

Reply:
ah, good catch! sure, I will make the change